### ISSUE TITLE
Setting completion status DLC & Update indication

### ISSUE THREAD

**Issue by official-markus-maurer** [h-quer/Slaanesh#33]
<img width="1369" alt="image" src="https://github.com/h-quer/Slaanesh/assets/33106127/5798796f-c954-4a07-b72a-2f5e1e307564">

I would appreciate a game version field when i add a completion date to a game, as well as a comment field which lets me write down which DLC i might have finished. Currently i can only add the version of the game i am playing in the general comment field of said game.

**Comment by h-quer** [comment]
Thanks for the interest in Slaanesh!
I'd like to have the ability to add custom columns to games / playthroughs eventually, that would cover your use case. This will take a while to implement, though.
In the meantime, maybe this helps for your use case: IGDB usually has individual entries for each DLC / expansion, e.g. [The Witcher main game](https://www.igdb.com/games/the-witcher-3-wild-hunt) and [Blood and Wine expansion](https://www.igdb.com/games/the-witcher-3-wild-hunt-blood-and-wine). What I do is add the main game and each expansion separately and then mark the main game or an expansion as completed individually. The comment field is then free for general use.

**Comment by RyuuSlayer** [comment]
Yeah by far the best backlog tool i have tried, and i tried many. Might as well be patient for a proper implementation of a DLC field, maybe something like a column with a little (+) allowing to add multiple ones for 1 game. Just thinking about there being over 2k nintendo switch DLC and like 100 of those being IGDB entries, doesn't make sense to currently add them as games :)

**Comment by Th3Rom3** [comment]
First of all: Thanks for maintaining this piece of software. This was exactly what I was looking for so long after several of the online resources shut down or changed beyond recognition over the last years.

I am sure this is not 100% what you were looking for but I found a quirk with the way data is handled right now which might get you a bit along the way:

If you use the exact same string for a label in both "Playing" and "Played positive" the game will show up both in "Playing" with its general game comment, as well as with the playthrough comments in "Played". I decided to use this for some games I replay with DLC.

![image](https://github.com/user-attachments/assets/178fd92c-fce1-4b16-a467-f9a50f241cf9)
![image](https://github.com/user-attachments/assets/adb1973b-64a6-4125-bc47-eae1f55d3b82)

Of course this behaviour might break if the way the filtering is done changes in the future but for now it is a neat workaround that does what I want it to do (i.e. have playthrough data but still keep it in "Playing" for DLC playthroughs).

**Comment by h-quer** [comment]
Neat, I didn't really consider how the game / playthrough comments would work with that separation.

Being able to have a game in two sections is intentional. It's also featured as an example in the wiki: https://github.com/h-quer/Slaanesh/wiki/Documentation-(sort-of)#status-and-platform-categories - no need to worry, this sorting behaviour isn't going to change any time soon.

**Comment by h-quer** [comment]
This should be covered with the neweset update and full UI/database rework, closing as completed.
